tests/test_cli.py

```diff
@@ -72,3 +72,31 @@ def test_aws_regions_offline(mock_aws_regions, runner):
     assert isinstance(parsed, list)
     assert "us-east-1" in parsed
     assert result.exit_code == 0
+
+@pytest.mark.e2e
+def test_gcp_images_live(runner):
+    """Run a live test against the Google Cloud API to get images via CLI."""
+    result = runner.invoke(cli.gcp_images)
+    parsed = json.loads(result.output)
+
+    assert isinstance(parsed, list)
+
+    for image in parsed:
+        expected_keys = ["status"]
```

Comment:
Does it make sense? It looks so empty :D

Reply:
That would be a bit restrictive indeed :D 
Probably my fault, should have added more information. 
So you currently test if all your keys are within the expected keys which would fail on live data as the returned data contains more than just the status key. What would make more sense at this point is checking if the keys that are returned contain the key that we want to find in there ("status") with the expected value (not "DEPRECATED").
Something similar to this could work:

`assert {image["UsageOperation"] for image in parsed} != "DEPRECATED"`

As soon as we start extracting the necessary information from the raw data this could be extended to test for all the keys we actually need for the rhelocator. I hope this makes kind of sense!

Reply:
Yes, made sense to me! Thank you again :)